<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="3" failures="0" skipped="0" tests="3" time="16.958" timestamp="2025-09-30T20:46:00.463315" hostname="Subarashi"><testcase classname="" name="tests.unit.test_auth_models" time="0.000"><error message="collection failure">ImportError while importing test module 'C:\Users\wesle\OneDrive\Documents\RealtorPro AI\Realtor-assistant\tests\unit\test_auth_models.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.13_3.13.2032.0_x64__qbz5n2kfra8p0\Lib\importlib\__init__.py:88: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests\unit\test_auth_models.py:15: in &lt;module&gt;
    from auth.utils import hash_password
backend\auth\utils.py:12: in &lt;module&gt;
    from email_validator import validate_email, EmailNotValidError
E   ModuleNotFoundError: No module named 'email_validator'</error></testcase><testcase classname="" name="tests.unit.test_auth_utils" time="0.000"><error message="collection failure">ImportError while importing test module 'C:\Users\wesle\OneDrive\Documents\RealtorPro AI\Realtor-assistant\tests\unit\test_auth_utils.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.13_3.13.2032.0_x64__qbz5n2kfra8p0\Lib\importlib\__init__.py:88: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests\unit\test_auth_utils.py:14: in &lt;module&gt;
    from auth.utils import (
backend\auth\utils.py:12: in &lt;module&gt;
    from email_validator import validate_email, EmailNotValidError
E   ModuleNotFoundError: No module named 'email_validator'</error></testcase><testcase classname="" name="tests.unit.test_basic" time="0.000"><error message="collection failure">'security' not found in `markers` configuration option</error></testcase></testsuite></testsuites>